README.md

```diff
@@ -0,0 +1,157 @@
+
+First, create the `openshift-cost` project. This is where we are going to deploy our Operator.
+
+Before running the operator, the CRD must be registered with the Kubernetes apiserver:
+
+```
+make install
+```
+
+Once this is done, there are two ways to run the operator:
+
+- As Go program outside a cluster
+- As a Deployment inside a Kubernetes cluster
+
+## Configuring your test environment
+
+Projects are scaffolded with unit tests that utilize the [envtest](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/envtest)
+library, which requires certain Kubernetes server binaries be present locally.
+Installation instructions can be found [here][env-test-setup].
+
+### 1. Run locally outside the cluster
+
+To run the operator locally execute the following command:
+
+```sh
+$ make run ENABLE_WEBHOOKS=false
+```
+
+### 2. Run as a Deployment inside the cluster
+
+#### Build and push the image
+
+Before building the operator image, ensure the generated Dockerfile references
+the base image you want. You can change the default "runner" image `gcr.io/distroless/static:nonroot`
+by replacing its tag with another, for example `alpine:latest`, and removing
+the `USER: nonroot:nonroot` directive.
+
+To build and push the operator image, use the following `make` commands.
+Make sure to modify the `IMG` arg in the example below to reference a container repository that
+you have access to. You can obtain an account for storing containers at
+repository sites such quay.io or hub.docker.com. This example uses quay.
+
+Build the image:
+```sh
+$ export USERNAME=<quay-username>
+
+$ make docker-build IMG=quay.io/$USERNAME/korekuta-operator-go:v0.0.1
+```
+
+Push the image to a repository:
+
+```sh
+$ make docker-push IMG=quay.io/$USERNAME/korekuta-operator-go:v0.0.1
+```
+
+**Note**:
+The name and tag of the image (`IMG=<some-registry>/<project-name>:tag`) in both the commands can also be set in the Makefile. Modify the line which has `IMG ?= controller:latest` to set your desired default image name.
+
+Branches of the repository are built automaticaly [here](https://quay.io/repository/project-koku/korekuta-operator-go).
+
+
+#### Deploy the operator
+
+
+```sh
+$ cd config/default/ && kustomize edit set namespace "openshift-cost" && cd ../..
+```
+
+Run the following to deploy the operator. This will also install the RBAC manifests from `config/rbac`.
+
+```sh
+$ make deploy IMG=quay.io/$USERNAME/korekuta-operator-go:v0.0.1
+```
+
+*NOTE* If you have enabled webhooks in your deployments, you will need to have cert-manager already installed
+in the cluster or `make deploy` will fail when creating the cert-manager resources.
+
+Verify that the korekuta-operator-go is up and running:
+
+```console
+$ oc get deployment
+```
+
+## Create a CostManagement CR
+
+Create the CR:
+
+```sh
+$ oc apply -f config/samples/cost-mgmt_v1alpha1_costmanagement.yaml
+```
+
+Ensure that the korekuta operator creates the deployment for the sample CR with the correct size:
```

Comment:
I don't think that this applies to us (I think it is for the Memcached Operator).

Reply:
you are right. I'll remove. Clearly didn't catch all the memcache references.

Reply:
done